src/win/tty.c

```diff
@@ -57,11 +57,17 @@
 
 static void uv_tty_capture_initial_style(CONSOLE_SCREEN_BUFFER_INFO* info);
 static void uv_tty_update_virtual_window(CONSOLE_SCREEN_BUFFER_INFO* info);
+static int uv_cancel_read_console(uv_tty_t* handle);
```

Comment:
Please make this `uv__cancel...`. I know the Windows side is not consistent with the rest, but let's try to make the new code consistent.